pkg/reconcilerevent/cluster.go

```diff
@@ -0,0 +1,36 @@
+package reconcilerevent
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+/*
+ These events are generated by the cluster operator.
+ */
+
```

Comment:
perhaps just wording, but this confused me for a second as it sounded like this file was code-generated

Reply:
I can use the term "produced"